core/src/Revolution/Processors/Search/Search.php

```diff
@@ -58,7 +55,15 @@ public function checkPermissions()
      */
     protected function getMaxResults()
     {
-        return $this->maxResults;
+        return intval($this->modx->getOption('fast_search_result_max', null, 10));
```

Comment:
Typecast via `(int)` works faster than function call. So, better to adjust it.

```suggestion
        return (int)$this->modx->getOption('fast_search_result_max', null, 10);
```